apps/srt-tunnel.cpp

```diff
@@ -450,6 +450,40 @@ class TcpMedium: public Medium
 
 #endif
 
+#ifdef _WIN32
+    static int tcp_send(int socket, const char* buf, int bufsize)
```

Comment:
The `send` and `recv` functions could be as well used on POSIX systems. Please replace existing implementation using `read` and `write` into `recv` and `send` respectively so that platform-dependent code is avoided.